content/en/docs/reference/controllers/job.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Job controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The {{< glossary_tooltip term_id="job" >}} controller creates 
+{{< glossary_tooltip term_id="pod" text="Pods" >}} to run each
+Job to completion.
+
+As its pods successfully complete, the controller tracks successful completions.
+When a specified number of successful completions is reached, the Job
+controller updates the Job object to mark it complete.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+The job controller is built in to kube-controller-manager.
+
+## Pod creation
```

Comment:
I feel the remainder of this information belongs in a new concepts/workloads/job.md document, actually. The vast majority of this information is about the Job object/concept and not specifically about the controller mechanism under the hood, which is what I think a reference/controllers/job.md document should focus on.